packages/cargo_nextest/project.bri

```diff
@@ -4,22 +4,21 @@ import rust, { cargoBuild } from "rust";
 export const project = {
   name: "cargo_nextest",
   version: "0.9.108",
-  repository: "https://github.com/nextest-rs/nextest.git",
+  extra: {
+    crateName: "cargo-nextest",
+  },
 };
 
-const source = Brioche.gitCheckout({
-  repository: project.repository,
-  ref: `cargo-nextest-${project.version}`,
-});
+const source = Brioche.download(
+  `https://crates.io/api/v1/crates/${project.extra.crateName}/${project.version}/download`,
+)
+  .unarchive("tar", "gzip")
+  .peel();
 
 export default function cargoNextest(): std.Recipe<std.Directory> {
   return cargoBuild({
     source,
-    path: "cargo-nextest",
-
-    // cargo-chef makes the lockfile out-of-date, see this issue:
-    // https://github.com/LukeMathWalker/cargo-chef/issues/305
-    cargoChefPrepare: false,
+    runnable: "bin/cargo-nextest",
```

Comment:
I also added `runnable`. It has nothing to do with the other changes of this PR.